.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: node_js
 
-node_js:
-  - 8
-  - 10
+env:
+  - FORCE_COLOR=true
 
-branches:
-  only: master
+node_js:
+  - 10.12
+  - 12
+  - node
```

Comment:
Why do you have to specify `node`? Isn't `10.12` and `12` enough?

Reply:
`node` is last active version, which at the moment is v13. With this we cover same versions as ESLint: 10.12, 12 and 13